src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends Figure {
+    public static final int FIGURE_COUNT = 5;
+    private static final Random random = new Random();
+    private static final Color WHITE = Color.WHITE;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(FIGURE_COUNT);
+        String color = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(DEFAULT_LENGTH) + 1;
+        switch (figureType) {
+            case 0 -> {
+                return new Circle(color,randomNumber);
+            }
```

Comment:
```suggestion
        return switch (figureType) {
            case 0 -> new Circle(color,randomNumber);
```
Make it beautiful :)